[rum] e-commerce guides (#16767)
DataDog/documentation: content/en/real_user_monitoring/guide/enable-rum-squarespace-store.md

---
title: Enable RUM on Your Squarespace Store
kind: guide
further_reading:
- link: '/real_user_monitoring/guide/rum-for-product-analytics/'
  tag: 'Documentation'
  text: 'Use RUM & Session Replay for Product Analytics'
- link: '/real_user_monitoring/guide/alerting-with-conversion-rates/'
  tag: 'Documentation'
  text: 'Alerting With Conversion Rates'
---

## Overview

> Understanding how customers interact with your web pages is crucial to the success of your online store. This guide will walk through how you can set up Real User Monitoring on your Squarespace-powered store.

Reviewer: ```suggestion
Understanding how customers interact with your web pages is crucial to the success of your online store.

This guide walks through how you can set up Real User Monitoring on your Squarespace-powered store.
```

## Setup

> 1. Log in your Squarespace administration panel and click on **Settings**.

Reviewer: ```suggestion
1. Log into your Squarespace administration panel and click **Settings**.
```

>    {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-1.png" alt="Enable RUM on your Squarespace store" style="width:40%;">}}

Reviewer: ```suggestion
   {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-1.png" alt="Enable RUM on your Squarespace store" style="width:30%;">}}
```

> 2. Under **Settings**, click on **Advanced**.

Reviewer: ```suggestion
2. Under **Settings**, click **Advanced**.
```

>    {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-2.png" alt="Enable RUM on your Squarespace store" style="width:40%;">}}

Reviewer: ```suggestion
   {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-2.png" alt="Enable RUM on your Squarespace store" style="width:30%;">}}
```

> 3. This will open up a new menu, click on **Code Injection**.

Reviewer: ```suggestion
3. In the open menu, click **Code Injection**.
```

>    {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-3.png" alt="Enable RUM on your Squarespace store" style="width:40%;">}}

Reviewer: ```suggestion
   {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-3.png" alt="Enable RUM on your Squarespace store" style="width:30%;">}}
```

> 4. Initialise the Browser RUM SDK by adding the SDK code snippet inside the **Header** section. For more information about which installation method to choose, you can refer to the [RUM Browser Monitoring documentation][1].

Reviewer: ```suggestion
4. Initialize the Browser RUM SDK by adding the SDK code snippet inside the **Header** section. See more information about which installation method to choose in the [RUM Browser Monitoring documentation][1].
```

      sessionSampleRate: 100,
      sessionReplaySampleRate: 100, // if not included, the default is 100
      trackResources: true,
      trackLongTasks: true,
      trackUserInteractions: true,
    });
  window.DD_RUM &&
    window.DD_RUM.startSessionReplayRecording();
</script>
```
{{</ site-region>}}
{{< site-region region="gov" >}}
```html
<script src="https://www.datadoghq-browser-agent.com/datadog-rum-v4.js" type="text/javascript"></script>
<script>
  window.DD_RUM &&
    window.DD_RUM.init({
      clientToken: '<CLIENT_TOKEN>',
      applicationId: '<APPLICATION_ID>',
      site: 'ddog-gov.com',
      //  service: 'my-web-application',
      //  env: 'production',
      //  version: '1.0.0',
      sessionSampleRate: 100,
      sessionReplaySampleRate: 100, // if not included, the default is 100
      trackResources: true,
      trackLongTasks: true,
      trackUserInteractions: true,
    });
  window.DD_RUM &&
    window.DD_RUM.startSessionReplayRecording();
</script>
```
{{</ site-region>}}
{{% /tab %}}
{{< /tabs >}}
  
   {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-4.png" alt="Enable RUM on your Squarespace store" >}}

> 5. Save the changes by clicking on the **Save** button.

Reviewer: ```suggestion
5. Click the **Save** button to save your changes.
```

>    {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-5.png" alt="Enable RUM on your Squaresapce store" style="width:60%;">}}

Reviewer: ```suggestion
   {{< img src="real_user_monitoring/guide/enable-rum-squarespace-store/enable-rum-squarespace-5.png" alt="Enable RUM on your Squarespace store" style="width:50%;">}}
```

> If needed, more information regarding code injection can be found in [Squarespace's documentation][2].

Reviewer: ```suggestion
See more information regarding code injection in [Squarespace's documentation][2].
```

## Start exploring

> You can now start using Real User Monitoring with your Squarespace store. Get valuable insights about your customers behavior, make data-driven decisions to improve your store, and increase conversion by watching browser recordings enriched sessions with [Session Replay][3], building [funnel analysis][4] to better understand customer journey, or [generating metrics][5] from those newly captured sessions.

Reviewer: ```suggestion
Once you've initialized the RUM Browser SDK, you can start using Real User Monitoring with your Squarespace store.

For example, you can:

- Get valuable insights about your customers' behavior by
making data-driven decisions to improve your store
- Increase conversion by watching browser recordings enriched sessions with [Session Replay][3]
- Use [funnel analysis][4] to better understand the customer journey, or 
- [Generate metrics][5] from those newly captured sessions
```
